cfme/configure/access_control.py

```diff
@@ -160,12 +177,20 @@ def create(self, cancel=False):
             'email_txt': self.email,
             'user_group_select': getattr(self.group, 'description', None)
         })
+
         if cancel:
             view.cancel_button.click()
             flash_message = 'Add of new User was cancelled by the user'
         else:
             view.add_button.click()
             flash_message = 'User "{}" was saved'.format(self.name)
+
+        try:
+            view.flash.assert_message(user_blocked_msg)
+            raise RBACOperationBlocked(user_blocked_msg)
```

Comment:
I actually quite like the idea of hooking this into the main exception handler, when we get any exception we first check to see if it was an RBACOperationBlock :)

Reply:
@psav wrote #5434 to capture this enhancement.